patch/cisco-add-support-for-ltc2979-chip.patch

```diff
@@ -0,0 +1,74 @@
+From 81db6d9122f00751609393dc4265c205215c22df Mon Sep 17 00:00:00 2001
+From: msiddare <msiddare@contoso.com>
+Date: Mon, 7 Jun 2021 20:07:57 +0000
+Subject: [PATCH] hwmon: (pmbus/ltc2978) Add support for new sensor
+
+Current ltc2978 driver does not have support for device ltc2979.
+
+Added support for the sensor device ltc2979.
+
```

Comment:
Please mention, what datasheet (name, revision) you used, and how it was tested.

Reply:
Updated patch with data sheet of chip and Test results.

Please review